src/features/formWizard/formWizard.tsx

```diff
@@ -0,0 +1,150 @@
+                  <FormErrorMessage>{error}</FormErrorMessage>
+                </FormControl>
+              )}
+            </Field>
+            <Field name="email" validate={zod2FieldValidator(emailSchema)}>
+              {({ input, meta: { error, touched } }) => (
+                <FormControl isInvalid={error && touched}>
+                  <Input {...input} type="email" placeholder="Email" />
+                  <FormErrorMessage>{error}</FormErrorMessage>
+                </FormControl>
+              )}
+            </Field>
+            <Field
+              name="password"
+              validate={zod2FieldValidator(passwordComplexity)}
+            >
+              {({ input, meta: { error, touched } }) => (
+                <FormControl isInvalid={error && touched}>
+                  <Input {...input} type="password" placeholder="Password" />
+                  <FormErrorMessage>{error}</FormErrorMessage>
+                </FormControl>
+              )}
+            </Field>
+          </Stack>
+        </WizardPage>
+        <WizardPage>
+          <Stack spacing={4}>
+            <Field name="phone" validate={zod2FieldValidator(phoneSchema)}>
+              {({ input, meta: { error, touched } }) => (
+                <FormControl isInvalid={error && touched}>
+                  <Input {...input} type="text" placeholder="Phone" />
+                  <FormErrorMessage>{error}</FormErrorMessage>
+                </FormControl>
+              )}
+            </Field>
+            <Field
+              name="billingAddress"
+              validate={zod2FieldValidator(required)}
+            >
+              {({ input, meta: { error, touched } }) => (
+                <FormControl isInvalid={error && touched}>
+                  <Input {...input} type="text" placeholder="Billing Address" />
+                  <FormErrorMessage>{error}</FormErrorMessage>
+                </FormControl>
+              )}
+            </Field>
+            <Field
+              name="shippingAddress"
+              validate={zod2FieldValidator(required)}
+            >
+              {({ input, meta: { error, touched } }) => (
+                <FormControl isInvalid={error && touched}>
+                  <Input
+                    {...input}
+                    type="text"
+                    placeholder="Shipping Address"
+                  />
+                  <FormErrorMessage>{error}</FormErrorMessage>
+                </FormControl>
+              )}
+            </Field>
+          </Stack>
+        </WizardPage>
+        <WizardPage>
+          <Stack spacing={4}>
+            <Field name="newsletter" type="checkbox">
+              {({ input, meta: { error, touched } }) => (
+                <CharkaCheckBoxFinalForm
+                  name="newsletter"
+                  label="Subscribe to Newsletter"
+                />
+              )}
+            </Field>
+            <Field name="specialOffers" type="checkbox">
```

Comment:
I don't understand the need of this nesting with the component function.

<CharkaCheckBoxFinalForm name="specialOffers" label="Receive Special Offers via Email" /> 

should suffice. I don't understand the need for `<Field>` and the the function given that neither `input` nor `meta` are used.

Reply:
@AndreaCuneo @AndreaLandiArk Field, input and meta Should not be needed for the checkboxes

Was confused as the config table was working for me but I'm on light